tests/data/MerchFactory.ts

```diff
@@ -38,22 +53,37 @@ export class MerchFactory {
     return substitutes.map((sub) => MerchandiseItemOptionModel.merge(MerchFactory.fakeOption(), sub));
   }
 
+  public static optionMetadataWith(...substitutes: Partial<MerchItemOptionMetadata>[]): MerchItemOptionMetadata[] {
+    return substitutes.map((sub) => {
+      const metadata = MerchFactory.fakeOptionMetadata();
+      return {
+        type: sub.type ?? metadata.type,
+        value: sub.value ?? metadata.value,
+        position: sub.position ?? metadata.position,
+      };
+    });
+  }
+
   public static fakeCollection(substitute?: Partial<MerchandiseCollectionModel>): MerchandiseCollectionModel {
     const fake = MerchandiseCollectionModel.create({
       uuid: uuid(),
       title: faker.datatype.hexaDecimal(10),
       description: faker.lorem.sentences(2),
+      themeColorHex: faker.internet.color(),
```

Comment:
We should specify somewhere that it's plain hex so people don't try sending it with the pound sign or `0x` or something.

Reply:
It actually does include the pound sign, but yes I'll include it in the Postman docs